hw_11/Controllers/Controller.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace hw_11.Controllers
+{
+    public abstract class Controller<TEntity>
```

Comment:
a bit strange name

Reply:
Do you remember a rule - if all method into abstract class are abstract - just use Interface instead